openmls/src/tree/tests/mod.rs

```diff
@@ -20,6 +20,8 @@ mod test_resolution;
 #[cfg(test)]
 mod test_secret_tree;
 #[cfg(test)]
+mod test_tree_truncation;
+#[cfg(test)]
```

Comment:
ugh, these `cfg` need to get cleaned up. This should get split into a `test-utils` and `test` module. Please file a follow up